.github/docker/Dockerfile.distroless-java-debian

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/distroless/java17-debian12:latest@sha256:8f80873debfafc0b77dd22d03e6d34d0a716c52404ca4ca19f76f2de11000c55
```

Comment:
We currently have java11 and java17 in use - I think for the minimal places we run jar files we can safely upgrade to a newer JVM.